static-instrumenter/agent-instrumenter/src/main/java/io/opentelemetry/contrib/staticinstrumenter/advices/AgentAdvices.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.staticinstrumenter.advices;
+
+import io.opentelemetry.contrib.staticinstrumenter.internals.Main;
+import java.lang.instrument.Instrumentation;
+import net.bytebuddy.asm.Advice;
+
+@SuppressWarnings({"SystemOut", "CatchAndPrintStackTrace"})
+public final class AgentAdvices {
```

Comment:
"Advice" is an uncountable noun -- WDYT about naming it `AgentAdviceClasses` instead?

Reply:
I tend to forget about this one, thanks for catching it!